react-components/src/query/useSearchMappedEquipmentAssetMappings.tsx

```diff
@@ -208,7 +208,7 @@ export const useMappingsForAssetIds = (
             nextCursor.model.revisionId === model.revisionId
         )?.cursor;
 
-        if (nextCursor === undefined) {
+        if (nextCursor === undefined || assetIds.length === 0) {
```

Comment:
useInfiniteQuery has no "enabled" option, right? Otherwise we could use it if the asset Ids is empty.
But I guess it doesnt have it, right?

Reply:
I think the code here is a better way to handle the situation, rather than disabling the query altogether. If the query is disabled, I think the result would be `undefined`, which we would then have to deal with as a special case somewhere else. It feels nicer to just return an empty list here as the query result, in my opinion